earth_enterprise/src/common/config/gePrompt.cpp

```diff
@@ -116,11 +116,11 @@ enterDirname(const std::string msg, const std::string &dflt,
 "%1 does not exist.\n"
 "Should this tool create it now")
                   .arg(dir), 'Y')) {
-        geCapabilitiesGuard cap_guard(
-            CAP_DAC_OVERRIDE,     // let me read all files
-            CAP_DAC_READ_SEARCH,  // let me traverse all dirs
-            CAP_CHOWN,            // let me chown files
-            CAP_FOWNER);          // let me chmod files I dont own
+        // geCapabilitiesGuard cap_guard(
```

Comment:
I saw in the commit description this was done so it would work in a non-privileged container. Do we want to keep it removed permanently?

Reply:
Is this an entirely secondary issue? I think it should be conditional, even if we temporary have it initially just under some compile-time flag.